frontend/components/Auction.vue

```diff
@@ -56,7 +63,7 @@
                                 <template v-if="auction.isActive && auction.marketUnitPriceToUnitPriceRatio">
                                     <format-market-value :value="auction.marketUnitPriceToUnitPriceRatio" />
                                 </template>
-                                <span v-else class="opacity-50">Unknown</span>
+                                <span v-else class="opacity-50">Not tradable</span>
```

Comment:
Since it's `v-else`, it will also be shown if `!auction.isActive`, which is not correct. `Not tradable` should only be shown when there is no `marketUnitPriceToUnitPriceRatio`